software/externals/swigmake

```diff
@@ -0,0 +1 @@
+Subproject commit ab03741a0627e99589ecbc1c088a4db05755e3c2
```

Comment:
Yes, although I think that might not be mandatory anymore after https://github.com/RobotLocomotion/drake/pull/1615 (@rdeits)?

But you might want the swig python bindings anyway (e.g. for the director) at some point.